src/main/scala/xerial/sbt/sonatype/SonatypeException.scala

```diff
@@ -20,7 +22,18 @@ object SonatypeException {
 
   case object MISSING_STAGING_PROFILE extends ErrorCode
 
-  case object MISSING_PROFILE extends ErrorCode
+  case class MISSING_PROFILE(profileName: String, host: String) extends ErrorCode {
+    val problem = s"Profile $profileName is not found on $host"
```

Comment:
```suggestion
    val problem = s"Profile ${profileName} is not found on ${host}"
```
Just for readability.